documentation/releases.md

```diff
@@ -6,21 +6,25 @@
 ## Supported versions
 
 | Version | Original Release Date | Latest Patch Version | Patch Release Date | End of Support | Runtime Frameworks |
-|---|---|---|---|---|---|
-| 7.1 | March 14th, 2023 | [7.1.0](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v7.1.0.md) | March 14, 2023 | | .NET 6<br/>.NET 7 |
-| 7.0 | November 11, 2022 | [7.0.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v7.0.2.md) | February 14, 2022 | June 14, 2023 | .NET 6<br/>.NET 7 |
-| 6.3 | October 11, 2022 | [6.3.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.3.2.md) | February 14, 2022 | | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
+| --- | --- | --- | --- | --- | --- |
+| 7.1 | March 14, 2023 | [7.1.0](https://github.com/dotnet/dotnet-monitor/releases/tag/v7.1.0) | March 14, 2023 |  | net6.0<br/>net7.0 |
+| 7.0 | November 11, 2022 | [7.0.2](https://github.com/dotnet/dotnet-monitor/releases/tag/v7.0.2) | February 14, 2023 | June 14, 2023 | net6.0<br/>net7.0 |
+| 6.3 | October 11, 2022 | [6.3.2](https://github.com/dotnet/dotnet-monitor/releases/tag/v6.3.2) | February 14, 2023 |  | netcoreapp3.1 (with major roll forward)<br/>net6.0 |
+
 
 ## Out of support versions
 
 | Version | Original Release Date | Latest Patch Version | Patch Release Date | End of Support | Runtime Frameworks |
-|---|---|---|---|---|---|
-| 6.2 | June 14, 2022 | [6.2.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.2.2.md) | August 9, 2022 | January 11, 2023 | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
-| 6.1 | February 17, 2022 | [6.1.4](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.1.4.md) | August 9, 2022 |  September 14, 2022 | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
-| 6.0 | November 8, 2021 | [6.0.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.0.2.md) | February 8, 2022 | May 17, 2022 | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
+| --- | --- | --- | --- | --- | --- |
+| 6.2 | June 14, 2022 | [6.2.2](https://github.com/dotnet/dotnet-monitor/releases/tag/v6.2.2) | August 9, 2022 | January 11, 2023 | netcoreapp3.1 (with major roll forward)<br/>net6.0 |
+| 6.1 | February 17, 2022 | [6.1.4](https://github.com/dotnet/dotnet-monitor/releases/tag/v6.1.4) | August 9, 2022 | September 14, 2022 | netcoreapp3.1 (with major roll forward)<br/>net6.0 |
+| 6.0 | November 8, 2021 | [6.0.2](https://github.com/dotnet/dotnet-monitor/releases/tag/v6.0.2) | February 8, 2022 | May 17, 2022 | netcoreapp3.1 (with major roll forward)<br/>net6.0 |
```

Comment:
Shouldn't this be dropped from the table since it's more than a year out of support?

Reply:
Sorry, read the wrong date.